unity-renderer/Assets/DCLPlugins/UUIDEventComponentsPlugin/UUIDComponent/PointerEventsController/PointerEventsController.cs

```diff
@@ -121,9 +122,17 @@ public void Update()
             }
 
             if (CollidersManager.i.GetColliderInfo(hitInfo.collider, out ColliderInfo info))
-                newHoveredInputEvent = info.entity.gameObject.GetComponentInChildren<IPointerEvent>();
+            {
+                // If an event exist in the new ECS, we got that value, if not it is ECS 6, so we continue as before
+                if (DataStore.i.ecs7.entityEvents.TryGetValue(info.entity.entityId, out List<IPointerInputEvent> pointerInputEvent))
```

Comment:
Let's wrap this in a descriptive function such as `IsECS7` or something like that.